test/tests/functional/pbs_node_rampdown.py

```diff
@@ -6919,6 +6949,8 @@ def test_release_multi_jobs(self):
         self.server.expect(JOB, {"exec_host": e_host_j3,
                                  "exec_vnode": e_vnode}, id=jid3)
 
+        self.logger.info("wait 10 secs all fib processes start")
+        time.sleep(10)
```

Comment:
instead of sleep please add a loop which will loop for 4-5 times with a sleep of 3-4sec.

Reply:
Thanks. please remove the sleep.